stackslib/src/core/tests/mod.rs

```diff
@@ -2611,6 +2625,7 @@ fn test_drop_and_blacklist_txs_by_size() {
             &mut chainstate,
             &ConsensusHash([0x1 + (block_height as u8); 20]),
             &BlockHeaderHash([0x2 + (block_height as u8); 32]),
+            false,
```

Comment:
Many uses of `false` -- can you add comments why these tests shouldn't try to resolve the nakamoto tenure?

Reply:
This is commented in the method docstring

Reply:
The comment in the docstring of the method just says that this parameter tells the function not to lookup nakamoto tenures. The tests that set this to false should say why they need that (because the chainstate is empty for the unit tests).